### PR TITLE
Jenkins setup ii

### PR DESCRIPTION
**Pull request by snocke**

### CODE REVIEW

File: kuttl-test/tests/spark/01-assert-minimalSpark.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 apiVersion: kuttl.dev/v1beta1
 kind: TestAssert
 commands:
-  - script: ./kuttl-test/tests/spark/sparkAssertion.sh
+  - script: /kuttl-test/tests/spark/sparkPreparation.sh
```
Comment (backstreetkiwi) [inline kuttl-test/tests/spark/01-assert-minimalSpark.yaml:4]: This relies on an absolute path. (2 occurences in this file)

File: .archiv/00-install-spark-operator.yaml [hunk 2]
```diff
@@ -4,3 +4,5 @@ commands:
   - command: helm repo add stackable-dev https://repo.stackable.tech/repository/helm-dev
   - command: helm repo update
   - command: helm upgrade spark-operator stackable-dev/spark-operator --version="0.5.0-nightly" --install
```
[the remark this answers is not in the document]
Comment (snocke) [reply .archiv/00-install-spark-operator.yaml:6]: This has been moved to .archiv and is not relevant for now. I haven't tested the local workflow at this point and therefore I didn't want to delete it just yet.

[the remark this answers is not in the document]
Comment (backstreetkiwi) [reply .archiv/00-install-spark-operator.yaml:6]: OK

File: .debug/startup.sh [hunk 3]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -x
+
+export HOST_WORKSPACE=$(pwd)
```
Comment (backstreetkiwi) [inline .debug/startup.sh:5]: You have to start this script from the top-level-folder of the repo for this to work properly, or am I missing something?

File: start-local-tests.sh [hunk 4]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -x
+
+REPO_DIR=$(dirname $0)
```
Comment (backstreetkiwi) [inline start-local-tests.sh:5]: I think the content of `$0` depends on how you run the command, right?

**Review by backstreetkiwi — APPROVED** [review]
I left two comments regarding the "debug"/local stuff, I didn't test that locally.
Jenkins is happy with the changes, so I'll approve. @snocke Please decide if you'd like another review from someone doing the tests locally.

File: .debug/startup.sh (hunk 3 above)
Comment (backstreetkiwi) [reply .debug/startup.sh:5]: We agreed that all stuff considering local startup is not relevant currently.

File: start-local-tests.sh (hunk 4 above)
Comment (backstreetkiwi) [reply start-local-tests.sh:5]: We agreed that all stuff considering local startup is not relevant currently.

**Pull request merged**